models/page_views/snowplow_page_views.sql

```diff
@@ -315,6 +316,29 @@ prep as (
       and a.domain_userid is not null
       and a.domain_sessionidx > 0
 
+),
+
+session_prep as (
+
+    select
+        *,
+        max(page_view_in_session_index) over (partition by session_id order by
```

Comment:
I think we could equivalently do this as a `count(*) over (...)` around line 112. My hunch is that Redshift would like that a little better, but it should work either way. Do you buy that?

Reply:
Yeah! I'll push it now.